src/Spryker/Zed/RabbitMq/Business/Model/Exchange/Exchange.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\RabbitMq\Business\Model\Exchange;
+
+use Psr\Log\LoggerInterface;
+use Spryker\Client\Queue\Model\Adapter\AdapterInterface;
+use Spryker\Zed\RabbitMq\Business\Model\Exchange\Filter\ExchangeFilterInterface;
+
+class Exchange implements ExchangeInterface
```

Comment:
I think it's better to rename `Exchange` and `Queue` classes to something else, related to deletion stuff, they have only deletion and purging?